### PR TITLE
Fix too many styling nodes

### PR DESCRIPTION
**Pull request by busypeoples**
This might need more work and testing. 

Includes ExtractTextPlugin setup  see #142

### CODE REVIEW

File: client/components/ComponentPreview.js [hunk 1]
```diff
@@ -18,7 +18,9 @@ class ComponentPreview extends React.Component {
   componentDidMount() {
     styleguideClientApi.on(this.state.componentPath, this.refreshComponentData);
     this.refreshComponentData();
-    styleguideClientApi.load(this.state.componentPath);
+    if (!has(styleguideClientApi.cache, this.state.componentPath)) {
+      styleguideClientApi.load(this.state.componentPath);
+    }
```
Comment (mxstbr) [inline client/components/ComponentPreview.js]: This is a brilliant fix for the whole too-many-styling-nodes issue I think, right?

**Comment by mxstbr** [comment]
This looks solid, but I haven't tried it yet. What do you think @nikgraf?

**Comment by busypeoples** [comment]
This needs a review and some refinements. @nikgraf try to run this and give me some feedback.

**Comment by nikgraf** [comment]
Reviewing now

**Comment by nikgraf** [comment]
uhh, what's the easiest way to run a branch from a fork?

**Comment by nikgraf** [comment]
@busypeoples I move this to here so I can test with the latest master: https://github.com/pure-ui/styleguide/pull/156

**Pull request closed without merging**